### PR TITLE
Ankush_Selenium_Assignment1

### PR DESCRIPTION
**Pull request by ghost**
Selenium Assignment1

### CODE REVIEW

File: seleniumByTechnocredits/src/seleniumPractise/ankush/AlertDemo.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+		String ActualMsg = driver.switchTo().alert().getText();
+		System.out.println(ActualMsg);
+
+		if (ActualMsg.equals("You must be TechnoCredits student!!")) {
+			driver.switchTo().alert().accept();
+			System.out.println("Alert accepted");
+		} else {
+			System.out.println("Wrong Message Displayed");
+		}
```
Comment (AmarjaBTechnocredits) [inline seleniumByTechnocredits/src/seleniumPractise/ankush/AlertDemo.java:41]: Alert is not accepted in else.Alert will remiain unattended if it goes in else.
Always close the browser as best practise.Add diver.close()

File: seleniumByTechnocredits/src/seleniumPractise/ankush/JavaScriptConfirmationMessage.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+		Thread.sleep(2000);
+		driver.switchTo().alert().dismiss();
+		String ActualCancelMessage = driver.findElement(By.id("pgraphdemo")).getText();
+
+		if (ActualCancelMessage.equals("You pressed Cancel!")) {
+			System.out.println("User Pressed on Cancel button");
+		}
+
+	}
+}
```
Comment (AmarjaBTechnocredits) [inline seleniumByTechnocredits/src/seleniumPractise/ankush/JavaScriptConfirmationMessage.java:51]: Code looks good.Add driver,close for all porgrams at the end

File: seleniumByTechnocredits/src/seleniumPractise/ankush/JavaScriptPrompt.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+		driver.switchTo().alert().dismiss();
+
+		String cancelTextMsg = driver.findElement(By.id("pgraphdemo")).getText();
+
+		if (cancelTextMsg.equals("User cancelled the prompt.")) {
+			System.out.println("User clickec on Cancelled Prompt Button");
+		}
+
+	}
+}
```
Comment (AmarjaBTechnocredits) [inline seleniumByTechnocredits/src/seleniumPractise/ankush/JavaScriptPrompt.java]: Code looks good

Comment (AmarjaBTechnocredits) [reply seleniumByTechnocredits/src/seleniumPractise/ankush/JavaScriptPrompt.java]: Please format the code using ctrl+shift_F

File: seleniumByTechnocredits/src/seleniumPractise/ankush/RegistrationFormValidation.java [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+		if (RegistrationFormValidation.alertHandling(driver).equals("Success")) {
+			System.out.println("All Validation are passed Successfully");
+		} else
+			System.out.println("Validations Fails");
+
+	}
+
+	public static void main(String[] args) throws InterruptedException {
+		WebDriver driver = Init.initChromeDriver();
+		RegistrationFormValidation.registrationFormChecks(driver);
```
Comment (AmarjaBTechnocredits) [inline seleniumByTechnocredits/src/seleniumPractise/ankush/RegistrationFormValidation.java]: For all the else part in code we will get exception alert unhandled .Please handle the same.
Format the code .

**Review by AmarjaBTechnocredits — CHANGES_REQUESTED** [review]

**Review by AmarjaBTechnocredits — APPROVED** [review]

**Review by KrishnaKTechnocredits — APPROVED** [review]

**Pull request merged**